composer.json

```diff
@@ -19,38 +19,39 @@
         "erusev/parsedown": "^1.6",
         "sensio/framework-extra-bundle": "^6.1",
         "symfony/apache-pack": "^1.0",
-        "symfony/asset": "^5.2",
-        "symfony/console": "^5.2",
-        "symfony/dotenv": "^5.2",
-        "symfony/expression-language": "^5.2",
+        "symfony/asset": "^5.3",
+        "symfony/console": "^5.3",
+        "symfony/dotenv": "^5.3",
+        "symfony/expression-language": "^5.3",
         "symfony/flex": "^1.1",
-        "symfony/form": "^5.2",
-        "symfony/framework-bundle": "^5.2",
-        "symfony/intl": "^5.2",
-        "symfony/mailer": "^5.2",
+        "symfony/form": "^5.3",
+        "symfony/framework-bundle": "^5.3",
+        "symfony/intl": "^5.3",
```

Comment:
you should also add `symfony/polyfill-intl-icu` package as relying on `symfony/intl` to provide the polyfill is deprecated